vendor.conf

```diff
@@ -5,7 +5,7 @@ github.com/opencontainers/runtime-spec 29686dbc5559d93fb1ef402eeda3e35c38d75af4
 # Core libcontainer functionality.
 github.com/checkpoint-restore/go-criu v3.11
 github.com/mrunalp/fileutils ed869b029674c0e9ce4c0dfa781405c2d9946d08
-github.com/opencontainers/selinux v1.0.0-rc1
+github.com/opencontainers/selinux v1.2
```

Comment:
(not directly related to this PR); I noticed that these tags miss a trailing `.0`, and because of that are following the SemVer syntax (which may complicate using them with Go mod)

Reply:
I can add another .0 if necessary.  in the next version.

Reply:
Thanks! I noticed it earlier this week, and this PR made me remember that I saw that 🤗